pkg/sql/zone_config.go

```diff
@@ -55,22 +55,17 @@ var errNoZoneConfigApplies = errors.New("no zone config applies")
 // descriptor in order to find its parent. If false, we'll assume that this
 // already is a parent and we'll not decode a descriptor.
 func getZoneConfig(
-	id config.SystemTenantObjectID,
+	id descpb.ID,
```

Comment:
[nit] Swap the order of arguments, codec typically precedes what's being decoded through it.